src/pytest_opentelemetry/instrumentation.py

```diff
@@ -75,6 +85,8 @@ def pytest_sessionfinish(self, session: Session) -> None:
         self.session_span.set_status(
             StatusCode.ERROR if self.has_error else StatusCode.OK
         )
+
+        self.try_force_flush()
         self.session_span.end()
```

Comment:
Would we want to flush _after_ the overall session's span is ended?  Like flip the order of these lines?

Reply:
That does indeed make more logical sense. I have switched these around now 😀